src/main/java/org/jenkinsci/plugins/testrail/JUnit/JUnitResults.java

```diff
@@ -43,6 +49,19 @@ public class JUnitResults {
     //private String[] Files;
     private List<TestSuite> Suites;
 
+    private static JAXBContext getJAXBContext(Class<?>... classesToBeBound) throws JAXBException {
+        Thread t = Thread.currentThread();
+        ClassLoader orig = t.getContextClassLoader();
+        t.setContextClassLoader(RealJenkinsRule.Endpoint.class.getClassLoader());
+        try {
+            return JAXBContext.newInstance(classesToBeBound);
```

Comment:
I am not sure what you are trying to accomplish here, but this is not the way to do it. Did you mean to use `PluginManager.uberClassLoader`? Or what class loader are you trying to set as contextual and why?